modules/Quiz/resources/quiz.js

```diff
@@ -557,6 +557,10 @@
                     $(".close-button")
                     .click(function () {
                         interfaceElement.find(".open-question-textarea").val(holdAnswer);
+                        interfaceElement.find(".open-question-chars .chars").text(holdAnswer.length);
+                        if (holdAnswer.length !== 0) {
+                            interfaceElement.find("#open-question-clear,#open-question-save").removeAttr("disabled");
```

Comment:
@shalommeoded - looks better. 
Have you tested that also when you visit an answer that was already submitted? (like  when you are reviewing the answer and cannot re-answer it)

Reply:
Yes and also when you have already answered, the UI will change from `Clear` and `Submit` to
`Change answer`